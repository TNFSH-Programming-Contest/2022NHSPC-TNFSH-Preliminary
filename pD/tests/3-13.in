8 7
1 2 1 0 2 1 0 4
2 1 1 2 2 1 2
.....T.
..TT...
......T
T......
...T..T
.......
....T.T
.T.T...
