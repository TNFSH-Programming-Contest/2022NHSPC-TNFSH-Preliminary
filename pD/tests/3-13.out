....AT.
.ATT..A
...A..T
T......
A..T.AT
...A...
....T.T
ATATA.A
